src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -86,10 +86,65 @@ class BaseController(object):
     Base class for all controllers providing API endpoints.
     """
     _cp_config_default = {
-        'request.error_page': {'default': json_error_page},
+        'request.error_page': {'default': json_error_page}
     }
 
 
+class RequestLoggingTool(cherrypy.Tool):
```

Comment:
Instead of building a new Cherrypy Tool, what do you think of making us of CherryPy's own access log handler? Have a look at this [example](https://stackoverflow.com/a/26975707/3185053).

Reply:
this is a particular case for logging since we are going to log into the ceph-mgr log. We are also logging different requests in different levels depending on their status codes plus logging the special case of start and end of a reques. For the above reasons I don't know how to fit the cherrypy log handler mechanisms.